Bottom: 39f80103c7ed1e6dd3ba3b39a722d5650e408a13
Top:    030710037ff423b76f8ff8badf1ba0d9135e98db
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-07-12 14:13:56 -0500

Refresh of known-hosts

---
diff --git a/ansible-playbook/buildbot.yml b/ansible-playbook/buildbot.yml
index 63cebaa2..bc6c7663 100644
--- a/ansible-playbook/buildbot.yml
+++ b/ansible-playbook/buildbot.yml
@@ -9,7 +9,7 @@
   - buildbot
   pre_tasks:
   - name: Scan Gitlab ssh keys
-    command: ssh-keyscan {{git_hostname}}
+    command: ssh-keyscan {{gitlab_hostname}}
     register: gitlab_keys
   roles:
   - name: Build and start the Buildbot master container
